test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -387,20 +389,28 @@ public void setupActiveAndStandbyCorfu() throws Exception {
         }
     }
 
-    public void openMaps(int mapCount) throws Exception {
+    public void openMaps(int mapCount, boolean diskBased) throws Exception {
         mapNameToMapActive = new HashMap<>();
         mapNameToMapStandby = new HashMap<>();
 
         for(int i=1; i <= mapCount; i++) {
             String mapName = TABLE_PREFIX + i;
 
+            Path pathActive = null;
+            Path pathStandby = null;
```

Comment:
nit: these 2 variables can be declared out of the for loop

Reply:
Done, thanks!